orchestrate.py

```diff
@@ -232,7 +233,7 @@ def main():
         (r"/spawn/?(/user/\w+(?:/.*)?)?", SpawnHandler),
         (r"/api/spawn/", APISpawnHandler),
         (r"/(user/\w+)(?:/.*)?", LoadingHandler),
-        (r"/stats", StatsHandler),
```

Comment:
Now that I'm thinking about it, we should probably have both /stats and /api/stats use the APIStatsHandler while also changing the tmpnb redirector.

Then folks can do migrations (ourselves included). After the deployments we know about have migrated (if they want to), then we'll change /stats to the new page you're working on.